packages/shared/src/db/queries.ts

```diff
@@ -3295,9 +3297,20 @@ export const insertActivityEvents = createWriteQuery(
           .values(activityEventGroups)
           .onConflictDoNothing();
       }
+
+      if (activityEventChannels.length) {
+        await Promise.all(
+          activityEventChannels.map((channelId) => {
+            return txCtx.db
+              .update($channels)
+              .set({ currentUserIsMember: true })
+              .where(eq($channels.id, channelId));
+          })
+        );
+      }
     });
   },
-  ['activityEvents']
```

Comment:
It probably should, yes. I wonder how that impacts performance

Reply:
it seems to work without it, so I think I'll leave like this for now. I'd rather not introduce perf issues.